UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MatrixSpawner.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using Tiles;
+using UnityEngine;
+
+public class MatrixSpawner : MonoBehaviour
```

Comment:
namespace

Reply:
Where is the namespace tool on rider? I can't find it